packages/sit-onyx/src/composables/useRipple.ts

```diff
@@ -7,9 +7,7 @@ export type RippleInstance = {
   animationEnded: boolean;
 };
 
-export const useRipple = (
-  container: Ref<Pick<HTMLElement, "getBoundingClientRect"> | undefined>,
-) => {
+export const useRipple = (container: Readonly<ShallowRef<HTMLElement | null>>) => {
```

Comment:
```suggestion
export const useRipple = (container: Readonly<ShallowRef<Pick<HTMLElement, "getBoundingClientRect"> | null>>) => {
```

There is a type error in the test file :)